src/.gitignore

```diff
@@ -33,8 +33,8 @@ src/react-app/coverage
 thunder-tests/
 
 # env files
-src/.env.*
-src/.env
+# src/.env.*
+# src/.env
```

Comment:
We still need to exclude .env files if that is the filename we're using across the different environments